lib/boat.rb

```diff
@@ -0,0 +1,20 @@
+class Boat
+
+  attr_reader :coords_hash
```

Comment:
My preference is not to use things like "hash" or "array" in the names of variables, as this is an implementation detail. I prefer variable names to be descriptive of what it is, not how its implemented.